java.lang.RuntimeException: An error occurred while executing doInBackground()
    at android.os.AsyncTask$3.done(AsyncTask.java:309)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
    at java.util.concurrent.FutureTask.run(FutureTask.java:242)
    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
    at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Boolean fr.gaulupeau.apps.Poche.entity.Article.getFavorite()' on a null object reference
    at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.processFeed(UpdateFeedTask.java:347)
    at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.updateByFeed(UpdateFeedTask.java:190)
    at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.updateAllFeeds(UpdateFeedTask.java:124)
    at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.doInBackground(UpdateFeedTask.java:71)
    at fr.gaulupeau.apps.Poche.network.tasks.UpdateFeedTask.doInBackground(UpdateFeedTask.java:30)
    at android.os.AsyncTask$2.call(AsyncTask.java:295)
    at java.util.concurrent.FutureTask.run(FutureTask.java:237)